docs/python/data-science-tutorial.md

```diff
@@ -0,0 +1,250 @@
+
+   ![Creating a new Jupyter Notebook](images/data-science-tutorial/notebook-editor.png)
+
+## Prepare the data
+
+This tutorial uses the [Titanic dataset](http://biostat.mc.vanderbilt.edu/wiki/pub/Main/DataSets/titanic.html) available on [OpenML.org](https://www.openml.org/d/40945), which is obtained from Vanderbilt University's Department of Biostatistics at [http://biostat.mc.vanderbilt.edu/DataSets](http://biostat.mc.vanderbilt.edu/DataSets). The Titanic data provides information about the survival of passengers on the Titanic, as well as characteristics about the passengers such as age and ticket class. Using this data, the tutorial will establish a model for predicting whether a given passenger would have survived the sinking of the Titanic. This section shows how to load and manipulate data in your Jupyter notebook.
+
+1. To begin, download the Titanic data from [OpenML.org](https://www.openml.org/d/40945) as a csv file named **data.csv** and save it to the **hello_ds** folder that you created in the previous section.
+1. In VS Code, open the **hello_ds** folder and the Jupyter notebook (**hello.ipynb**), by going to **File** > **Open Folder**.
+1. Within your Jupyter notebook begin by importing the [pandas](https://pandas.pydata.org/) and [numpy](https://numpy.org/) libraries, two common libraries used for manipulating data, and load the Titanic data into a pandas [DataFrame](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html). To do so, copy the below code into the first cell of the notebook. For additional guidance about working with Jupyter notebooks in VS Code, see the documentation [here](https://code.visualstudio.com/docs/python/jupyter-support).
+
+   ```python
+   import pandas as pd
+   import numpy as np
+   data = pd.read_csv('data.csv')
+   ```
+
+1. Now, run the cell using the Run cell icon.
+
+   ![Running a Jupyter notebook cell](images/data-science-tutorial/jupyter-cell-01.png)
+
+1. After the cell finishes running, you can view the data that was loaded using the variable explorer and data viewer. First click on the chart icon in the notebook's upper toolbar, then the data viewer icon to the right of the *data* variable. For additional information about the data set, refer to [this document](http://biostat.mc.vanderbilt.edu/wiki/pub/Main/DataSets/titanic3info.txt) about how it was constructed.
+
+   ![Data viewer and variable explorer](images/data-science-tutorial/data-viewer.png)
+
+   You can then use the data viewer to view, sort, and filter the rows of data. After reviewing the data, it can then be helpful to graph some aspects of it to help visualize the relationships between the different variables.
+
+1. Before the data can be graphed though, you need to make sure that there aren’t any issues with it. If you look at the Titanic csv file, one thing you’ll notice is that a question mark (“?”) was used to designate cells where data wasn’t available.
```

Comment:
Maybe we can add a screenshot of the data viewer and also get the user to try to use the Dataviewer filter to look for '?' in the columns

Reply:
Agree with you, but leaving out for now. I want to verify how our name usage guidelines apply to something like this first and don't want to block on it